ResearchKit/ActiveTasks/ORKAccelerometerRecorder.m

```diff
@@ -192,18 +190,27 @@ @implementation ORKAccelerometerRecorderConfiguration
 
 #pragma clang diagnostic push
 #pragma clang diagnostic ignored "-Wobjc-designated-initializers"
-- (instancetype)initWithFrequency:(double)freq {
-    self = [self ork_init];
+
+- (instancetype)initWithIdentifier:(NSString *)identifier {
```

Comment:
If we move `ORKRecorderConfiguration` `initWithIdentifier:(NSString *)identifier` from the *public* to the *internal* header (as per you suggestion in the comment below), do we still need to mark these as `NS_UNAVAILABLE`? They will just not be visible from outside the framework.